.github/workflows/go.yml

```diff
@@ -18,14 +18,20 @@ jobs:
       with:
         go-version: stable
 
-    - name: Run tests in maven
-      working-directory: ./util/maven
-      run: go test ./...
+    - name: Build examples
+      working-directory: examples/go/
+      run: |
+        for dir in $(ls -d */); do
+          cd "$dir"
+          go build
+          cd ..
+        done
 
-    - name: Run tests in resolve
-      working-directory: ./util/resolve
-      run: go test ./...
-
-    - name: Run tests in semver
-      working-directory: ./util/semver
-      run: go test ./...
+    - name: Run util tests
+      working-directory: util/
+      run: |
```

Comment:
Same here:

    run: find . -name 'go.mod' -execdir go test \;

Reply:
I think we need to run `go test ./...` for `util/resolve` considering there are per-ecosystem tests